docs/architecture/smart-rollups.md

```diff
@@ -179,9 +179,9 @@ When there is only one commitment left, either because all nodes published ident
 The commitments for the next commitment period build on the last cemented commitment.
 
 The refutation period lasts for a set number of blocks based on the `smart_rollup_challenge_window_in_blocks` protocol constant.
-For example, currently the refutation period lasts 120,960 blocks on Mainnet.
-Mainnet has 10 seconds between blocks as of the Paris protocol upgrade, which means that the refutation period lasts 2 weeks.
-Ghostnet has 5 seconds between blocks but its refutation period is twice as many blocks, so its refutation period is also two weeks long.
+For example, currently the refutation period lasts 151,200 blocks on Mainnet.
```

Comment:
I didn't check that one. How did you get the info?

Reply:
I calculate the refutation period as `(consensus_rights_delay + 2) * blocks_per_cycle * minimal_block_delay`. Somewhere I had a link to those values for mainnet Quebec but can't find them at the moment. Maybe just "two weeks" is better here.